Target-MVD-Mathias/Extensions/StringExtension.swift

```diff
@@ -34,4 +34,14 @@ extension String {
     return first + other
   }
 
+  func isValidUsername() -> Bool {
+    let whitespaceSet = NSCharacterSet.whitespaces
+    let username = self.trimmingCharacters(in: whitespaceSet)
```

Comment:
have you tried to use .whitespaceSet directly? or why not use NSCharacterSet.whitespaces directly, WDYT?